tests/test_voting_classifier.py

```diff
@@ -0,0 +1,139 @@
+from sklearn.utils.validation import check_is_fitted
+import numpy as np
+
+from wellcomeml.ml import WellcomeVotingClassifier
+
+
+class MockEstimator():
+    def __init__(self, Y_prob, multilabel=False):
+        self.fittted_ = True
+        self.Y_prob = Y_prob
+        self.multilabel = multilabel
+        self.Y = self.get_Y()
+
+    def get_Y(self):
+        if self.multilabel:
+            Y = self.Y_prob > 0.5
+            return Y.astype(int)
+        else:
+            Y = np.argmax(self.Y_prob, axis=1)
+            return Y
+
+    def fit(self, X, Y):
+        pass
+
+    def predict(self, X):
+        return self.Y
+
+    def predict_proba(self, X):
+        return self.Y_prob
+
+def test_multilabel():
+    Y1_prob = np.array([
+        [0.9, 0.5],
+        [0.2, 0.8],
+        [0.3, 0.9]
+    ])
+    Y2_prob = np.array([
+        [0.7, 0.6],
+        [0.6, 0.5],
+        [0.4, 0.3]
+    ])
+    Y_expected = np.array([
+        [1, 1],
+        [0, 1],
+        [0, 1]
+    ])
+    est1 = MockEstimator(Y1_prob, multilabel=True)
+    est2 = MockEstimator(Y2_prob, multilabel=True)
+
+    voting_classifier = WellcomeVotingClassifier(
+        estimators=[est1, est2], voting="soft"
+    )
+    X = ["mock", "data", "not used"]
+    Y = voting_classifier.predict(X)
+    assert np.array_equal(Y, Y_expected)
+
+def test_multiclass():
```

Comment:
isnt this the `test_binary`?

Reply:
hmm, i am also confused. i think you are right. at this point i am not sure what the point of both this and the other test is but will change names accordingly.

Reply:
Resolved in 30fe1421487dc349826ed0a6d0fce782f31477e1